src/functions/findFirst.js

```diff
@@ -7,12 +7,27 @@ import {copyable} from "./copyable";
 import {equals} from "./equals";
 import {FindSequenceResult, ForwardFindSequenceThread, stepFindThreads} from "./findAll";
 
-// Find the first occurrence of a substring as judged by a comparison function.
-// Finding the first instance of a substring is overwhelmingly the most
-// common use case for substring searching, so alias "find" to "findFirst"
-// for maximum user convenience and minimum user confusion.
 export const findFirst = wrap({
     names: ["findFirst", "find"],
+    summary: "Finds the first occurrance of a substring as judged by a comparison function.",
+    docs: process.env.NODE_ENV !== "development" ? undefined : {
+        introduced: "higher@1.0.0",
+        detail: (`
+
+        `),
+        expects: (`
+            The function expects as input a comparison function and a sequence array
+            which should contain a source sequence and a bounded search sequence.
+
+            The function expects as input a sequence known to be bounded and
```

Comment:
Yeah I was going to eventually put something in here. Just emptied it out to remember that I hadn't done it yet :)